adal/tests/Test.ADAL.NET.UIAutomation/XamarinDroidADALTests.cs

```diff
@@ -124,6 +124,7 @@ public void AcquireTokenADFSv3FederatedInteractiveTest()
         /// Runs through the standard acquire token flow
         /// </summary>
         [Test]
+        [Ignore("Failing due to security policies.")]
```

Comment:
As @jmprieur commented, please create a VSTS bug to track this and add it as a comment next to the Ignore attribute. Otherwise, absolutely ignore failing tests to investigate, they are just noise.